src/uu/cp/cp.md

```diff
@@ -0,0 +1,12 @@
+# cp
+
+## About
+
+Copy SOURCE to DEST, or multiple SOURCE(s) to DIRECTORY.
+
+## Usage
+```
+cp [OPTION]... [-T] SOURCE DEST
+cp [OPTION]... SOURCE... DIRECTORY
+cp [OPTION]... -t DIRECTORY SOURCE...
+```
```

Comment:
Nit, but could you move this above the `## About` for consistency? See the discussion in https://github.com/uutils/coreutils/issues/4368 for context.

Reply:
I missed the discussion.
Now fixed.